.flake8

```diff
@@ -1,6 +1,6 @@
 [flake8]
 inline-quotes = "
-ignore = E203, E266, W503, ANN002, ANN003, ANN101, ANN102, ANN401, N807, N818
+ignore = E203, E266, W503, ANN002, ANN003, ANN101, ANN102, ANN401, N807, N818, F821
```

Comment:
don't change flake8 settings, try to rewrite ur code with avoiding of error

Reply:
Corrected

Reply:
attempt is not successful, need F281, otherwise conflict appending classes

Reply:
Fixing was so long, because try to solve this problem

Reply:
okey